core/windows_and_tabs/window_management.talon

```diff
@@ -17,5 +17,43 @@ snap last [screen]: user.move_window_previous_screen()
 snap screen <number>: user.move_window_to_screen(number)
 snap <user.running_applications> <user.window_snap_position>:
     user.snap_app(running_applications, window_snap_position)
+snap left <user.running_applications> <user.running_applications>:
```

Comment:
"snap left" doesn't feel like the right grammar for this, given that the entire screen will be filled by these two applications after the command, right? Given that you also have things like `snap clock`, this second word feels like a noun referring to the arrangement. It seems to me like you would want something like "snap side by side", only shorter?

Reply:
I considered "snap split" but you might want to focus on the right or the left split, which led me to think of "snap left" to mean "the main one should be to the left". Open to suggestions though.

Reply:
`snap split` sounds best to me.

> but you might want to focus on the right or the left split

Ah, that makes sense now that you point it out. Hm, I'm not sure either.

Reply:
I could see an argument for making `snap split` always focus the left split as a first pass at this; Maybe we'll learn something that will inform how it could work better.